staging/src/k8s.io/pod-security-admission/test/fixtures.go

```diff
@@ -65,21 +95,61 @@ func init() {
 			Type: corev1.SeccompProfileTypeRuntimeDefault,
 		}
 	})
-	minimalValidPods[api.LevelRestricted][api.MajorMinorVersion(1, 19)] = restricted_1_19
+	minimalValidOSNeutralPods[api.LevelRestricted][api.MajorMinorVersion(1, 19)] = restricted_1_19
+	minimalValidLinuxPods[api.LevelRestricted][api.MajorMinorVersion(1, 19)] = addLinux(restricted_1_19)
+	minimalValidWindowsPods[api.LevelRestricted][api.MajorMinorVersion(1, 19)] = addWindows(restricted_1_19)
 
 	// 1.22+: capabilities.drop=["ALL"]
 	restricted_1_22 := tweak(restricted_1_19, func(p *corev1.Pod) {
 		p.Spec.Containers[0].SecurityContext.Capabilities = &corev1.Capabilities{Drop: []corev1.Capability{"ALL"}}
 		p.Spec.InitContainers[0].SecurityContext.Capabilities = &corev1.Capabilities{Drop: []corev1.Capability{"ALL"}}
 	})
-	minimalValidPods[api.LevelRestricted][api.MajorMinorVersion(1, 22)] = restricted_1_22
+	minimalValidOSNeutralPods[api.LevelRestricted][api.MajorMinorVersion(1, 22)] = restricted_1_22
+	minimalValidLinuxPods[api.LevelRestricted][api.MajorMinorVersion(1, 22)] = addLinux(restricted_1_22)
+	minimalValidWindowsPods[api.LevelRestricted][api.MajorMinorVersion(1, 22)] = addWindows(restricted_1_22)
+
+	// 1.25+: OS specific changes
```

Comment:
I'd suggest capturing `addWindows(restricted_1_0)` in a variable with a comment, and so we have it as a starting point if any future versions add restricted requirements for windows pods

```go
// none of the restricted requirements added between 1.0 and 1.25 apply to pods that are explicitly Windows
restricted_1_25_windows := addWindows(restricted_1_0)
...
minimalValidWindowsPods[api.LevelRestricted][api.MajorMinorVersion(1, 25)] = restricted_1_25_windows
```

Reply:
this is still outstanding